examples/catalog/bin/sample_page.dart

```diff
@@ -136,7 +141,8 @@ class SampleInfo {
     commentValues['path'] = 'examples/catalog/${sourceFile.path}';
     commentValues['source'] = sourceCode.trim();
     commentValues['link'] = link;
-    commentValues['android screenshot'] = 'https://storage.googleapis.com/flutter-catalog/$commit/${sourceName}_small.png';
+    commentValues['android screenshot'] =
+        'https://storage.googleapis.com/flutter-catalog/$commit/${sourceName}_small.png';
```

Comment:
I believe there's a strong preference to not have things like this line broken.

Reply:
This PR use a 120 chars Line length. The break seems legit here with this setup.